script.artistslideshow/changelog.txt

```diff
@@ -0,0 +1,354 @@
+v.3.0.0
```

Comment:
please drop. Leia and up do not use the changelog in the interface but only the news tag. You're still free to keep it along your code in the upstream repo

Reply:
Will do.  Since I'm going to have to manually submit to delete the .gitignore file anyway, I can delete this one too.  I still greatly dislike the fact that the changelog interface was removed.  Having news in the addon is just double work for addon developers.  But I know that ship has sailed.